### ISSUE TITLE
Upload fails with a 0-byte file.

### ISSUE THREAD

**Issue by RubenGarcia** [rpldy/react-uploady#91]
**Describe the bug**
When I try to upload a 0-byte file, I get the following error:
```
Uncaught (in promise) ChunkedSendError: start byte 0 is invalid. File size: 0
```
**To Reproduce**
1. Install
    "@rpldy/tus-uploady": "^0.7.0",
    "@rpldy/uploader": "^0.7.0",
    "@rpldy/uploady": "^0.7.0",
2. Use code such as
const uploader = createUploader({ 
			enhancer: tusEnhancer,
    			autoUpload: true,
    			grouped: false,
			destination: {url: config.url.datasetTus,
				      headers: {Authorization: getAuthorization()}
			},
			sendDataOnCreate: true,
		  });
3. Check the console for Uncaught (in promise) ChunkedSendError: start byte 0 is invalid. File size: 0
**Expected behavior**
Correct upload of the 0-size file.

**Versions**
Specify the versions of the @rpldy packages you're using. 
    "@rpldy/tus-uploady": "^0.7.0",
    "@rpldy/uploader": "^0.7.0",
    "@rpldy/uploady": "^0.7.0",

Google chrome: Version 85.0.4183.121 (Offizieller Build) (32-Bit)

**Comment by yoavniran** [comment]
Hey @RubenGarcia 
will have a fix for this soon.
out of curiosity, is this a common issue? uploading empty file? whats the use-case?

**Comment by RubenGarcia** [comment]
If the users are left to upload their files, this is something that might happen. It is more user-friendly to have one upload button than two (tus vs normal upload). I discovered it because one of my files did not close correctly and I ended up with a 0-byte file which I tried to upload without noticing it was empty.

**Comment by yoavniran** [comment]
Got it. Thanks. Should be fixed now in 0.7.1
